kafka/trigger/README.md

```diff
@@ -0,0 +1,99 @@
+<!--
+title: Kafka
+weight: 4701
+-->
+# Kafka Trigger
+
+This trigger subscribes to a topic on Kafka cluster and listens for the messages.
+
+### Flogo CLI
+```bash
+flogo install github.com/project-flogo/messaging-contrib/kafka/trigger
+```
+
+## Configuration
+
+### Setting :
+
+| Name       | Type   | Description
+|:---        | :---   | :---     
+| connection | any    | The connection object which is use to connect to pulsar - ***REQUIRED***
```

Comment:
This shouldn't be pulsar.  Also, maybe have a reference to the connection readme.